flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/sql/ScalarSqlFunctions.scala

```diff
@@ -67,6 +67,14 @@ object ScalarSqlFunctions {
     OperandTypes.ONE_OR_MORE,
     SqlFunctionCategory.STRING)
 
+  val REPLACE = new SqlFunction(
```

Comment:
The `REPLACE` SqlFunction has been provided in `org.apache.calcite.sql.fun.SqlStdOperatorTable`.

Reply:
I use SqlStdOperatorTable.REPLACE at first, however that REPLACE function can not handle null params, so I create a new function which refer to SqlStdOperatorTable.REPLACE.